fastestimator/backend/feed_forward.py

```diff
@@ -62,7 +62,10 @@ def feed_forward(model: Union[tf.keras.Model, torch.nn.Module], x: Union[Tensor,
         model.train(mode=training)
         if not isinstance(x, torch.Tensor):
             x = to_tensor(x, "torch")
-        x = model(x)
+        if isinstance(x, list):
+            x = model(*x)
```

Comment:
what if someone actually wants to pass a list as input?

Reply:
done